homeassistant/components/husqvarna_automower/strings.json

```diff
@@ -22,5 +22,55 @@
     "create_entry": {
       "default": "[%key:common::config_flow::create_entry::authenticated%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "number_of_charging_cycles": {
+        "name": "Number of charging cycles"
+      },
+      "number_of_collisions": {
+        "name": "Number of collisions"
+      },
+      "cutting_blade_usage_time": {
+        "name": "Cutting blade usage time"
+      },
+      "total_charging_time": {
+        "name": "Total charging time"
+      },
+      "total_cutting_time": {
+        "name": "Total cutting time"
+      },
+      "total_running_time": {
+        "name": "Total running time"
+      },
+      "total_searching_time": {
+        "name": "Total searching time"
+      },
+      "searching_time_percent": {
+        "name": "Searching time percent"
+      },
+      "cutting_time_percent": {
+        "name": "Cutting time percent"
+      },
+      "total_drive_distance": {
+        "name": "Total drive distance"
+      },
+      "battery_level": {
+        "name": "Battery level"
+      },
+      "next_start_timestamp": {
+        "name": "Next start"
+      },
+      "mode": {
+        "name": "Mode",
+        "state": {
+          "main_area": "Main area",
+          "secondary_area": "Secondary area",
+          "home": "Home",
+          "demo": "Demo",
+          "unknown": "Unknown"
```

Comment:
Unknown shouldn't be a value the sensor should return. Instead, the sensor should return `None`. HA will then render it unknown.

Reply:
done in https://github.com/home-assistant/core/pull/110410/commits/63e53a206dc05a5fa27423df05ac7dedaf4b4ab2